### PR TITLE
Add proto definitions for P4Runtime and P4Info

### PR DESCRIPTION
**Pull request by antoninbas**
Copied from p4lang/PI

### CODE REVIEW

**Comment by antoninbas** [comment]
@chrispsommers added the protobuf IDLs

**Comment by chrispsommers** [comment]
@antoninbas It's good to get this in the repo. What are the implications of code under PI? Will it live on but migrate to referencing this repo? I guess all the APIs will have to change to the semantically-versioned APIs.

**Comment by antoninbas** [comment]
@wmohsin I updated the PR with the directory structure we discussed. PTAL.

File: v1/README.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# P4Runtime Specification Version 1
-
-This directory contains protobuf files, specifications and related artifacts for
```
Comment (chrispsommers) [inline v1/README.md]: Should we modify this text in the top-level README and possibly add some READMEs into the `proto` directory structure at suitable places?
> This directory contains protobuf files, specifications and related artifacts for all versions of the P4Runtime API. A separate directory is assigned to each major version, e.g. v1. Therein, tags are used to mark minor and patch release versions.

Comment (antoninbas) [reply v1/README.md]: Thanks for pointing this out, I'm going to update the README before I merge

**Pull request merged**